recipe/recipe.yaml

```diff
@@ -148,6 +149,13 @@ outputs:
         - xxhash
         - zlib
         - zstd
+        # Required for roofit_multiprocess
+        - cppzmq
+        # Required zmq_ppoll API added in v4.3.5
+        - zeromq
+        # Need to request drafts API
+        # c.f. https://github.com/conda-forge/root-feedstock/pull/292#issuecomment-3268262584
+        - zeromq * drafts_*
```

Comment:
I find the repetition of `zeromq` a bit puzzling, is the second line going to supersede the first one?

Reply:
If I have it right the logic is that `zeromq` is in the global pinning and so by asking for

```yaml
- zeromq
```

this updates the `zeromq` pinned version in the `.ci_support/` files. But as we need the drafts variant of `zeromq` then we need to also explicitly request that

```yaml
- zeromq * drafts_*
```

but also want to make sure that version is aligned with the global pinning, and so we need both together to get the correct versions for the build.

Reply:
Thanks very much for the clarification, @minrk! 

@vepadulano, is this clear now?

Reply:
> I recommend leaving a comment about the pinning next to it, so folks don't remove the apparently redundant dependency in the future (I've done this for similar weird workarounds for conda-build behavior that I forgot the reason for years later).

Updated the one I already had in to now point to the nice expletnation in https://github.com/conda-forge/root-feedstock/pull/292#discussion_r2334082389.